src/main.js

```diff
@@ -2,12 +2,10 @@ import { DocoffPlaceholder } from './DocoffPlaceholder';
 import { DocoffReactPreview } from './DocoffReactPreview';
 import { DocoffReactBase } from './DocoffReactBase';
 import { DocoffReactProps } from './DocoffReactProps';
-import { DocoffFunctionDoc } from './DocoffFunctionDoc';
 
 customElements.define('docoff-react-preview', DocoffReactPreview, { extends: 'textarea' });
 customElements.define('docoff-react-base', DocoffReactBase, { extends: 'textarea' });
 customElements.define('docoff-react-props', DocoffReactProps);
-customElements.define('docoff-function-doc', DocoffFunctionDoc);
```

Comment:
Lets keep this custom HTML element and if it is not replaced by the Webpack plugin it should show some warning that the file needs to be run through the build process first.